endpoints/openrtb2/auction_test.go

```diff
@@ -2076,6 +2077,245 @@ func TestValidateBidders(t *testing.T) {
 	}
 }
 
+func TestWorkaroundIOS14PrivacyBug(t *testing.T) {
```

Comment:
There's a LOT of possible test case combinations. I likely already included too many, but I cut myself off at testing all initial values of the LMT flag. In most cases, it's set to `nil` at the start. There are a few test cases which ensure the LMT is overridden when set to a different non-nil value.

Reply:
I very much disliked my original solution and rewrote it.